src/app/messaging/messaging.component.ts

```diff
@@ -17,41 +17,37 @@ export class MessagingComponent implements OnInit {
   constructor(private messageService: MessageService) { }
 
   displayMessageLimit: number = 10;
-  displayMessages: MessageModel[];
-  displayMessagesOffset: number = 0;
-  myMessages: MessageModel[];
-  otherMessages: MessageModel[];
-  request: MessageRequestModel;
-
-  inputMessage: NewMessageModel;
-
-  ngOnInit(): void {
-    this.request = this.generateRequest();
-    this.getMessagesToCurrentUser();
-    this.inputMessage = this.generateNewMessage();
-  }
-
-  getMessagesToCurrentUser(): void {
-    //TODO read current user Id from auth data
-    // this.messageService.getMessagesToRecipient(this.request)
-    //     .subscribe(
-    //       res => this.handleResponse(res),
-    //       err => alert("Error! " + err)
-    //     );
-
-    //TODO replace this with actual dialogue retrieveing mechanics
-    this.messageService.getDialogue(this.request)
-        .subscribe(
-          res => this.handleResponse(res),
-          err => alert("Error! " + err)
-        );
-  }
-
-  getMessagesFromTo(): void {
+  displayMessages: MessageResponseModel[];
+  displayMessagesOffset: number = 0;  
+
+  //TODO Fill with actual values!
+  getMessagesRequest: UserMessageRequestModel = {
+    messageType: 0,
```

Comment:
I believe you have enum for messageTypes.

Reply:
Corrected.